Dockerfile

```diff
@@ -1,6 +1,6 @@
 #FROM alpine:3.7
 ARG PROXY_IMAGE_URL=blank
-FROM ubuntu:18.04
+FROM ubuntu:16.04
```

Comment:
Any reason why is this reverted to ubuntu 16?

Reply:
Travis is failing it is not able to pull 18 or any other ubuntu image. Some travis and distro specific issue. Anyways we are not going to use this for managed env